### DISCUSSION TITLE
"HitHighlightedSummary" property is not returning the searched keywords with <c0> tag, else it returns only plain text in PNP modern Search v4.9.3.0

### DISCUSSION THREAD

**Discussion by krishnare56** [microsoft-search/pnp-modern-search#3548] [Q&A]
I am using PNP modern search V4 webparts in SharePoint online site. where i have linked search box and search results webpart. i need to highlight the Searched keyword in search results, i referred some documents where it stats "HitHighlightedSummary" will return the summery with highlighting the matched keywords, but in my case its simply returning the summery without highlighting the keywords. i am not sure what i am missing.

i have tried adding "HitHighlightedSummary" in Hit-Highlighted properties to but result is still same.

please find the below image for reference, where i have searched with word GCS but in "HitHighlightedSummary" it still returns plain summary.
<img width="566" alt="0kKpl" src="https://github.com/microsoft-search/pnp-modern-search/assets/131133647/5c3571dc-9c59-4aee-8342-1bbe0ea92939">

**Comment by wobba** [comment]
From the screenshot the highlight for this case is in the Title, and highlighted in HitHighlightedProperties. And any API behavior is at the mercy of what the API return. You can modify the display template to also show data from the other property to match your scenario.

**Reply by krishnare56** [reply]
But the expectations are the  "HitHighlitedSummery" properties need to return with highlighted text same has Hithighlitedproperties right. but it is not working has expected

**Comment by krishnare56** [comment]
raised an issue as i dint get much info in discussion, closing this discussion, please refer the issue link- https://github.com/microsoft-search/pnp-modern-search/issues/3562
